src/Ocis.php

```diff
@@ -306,6 +306,33 @@ private function getServiceUrlFromWebfinger(string $webfingerUrl): string
         }
         throw new InvalidResponseException('invalid webfinger response');
     }
+
+    /**
+     * returns current oCIS version
+     *
+     * @return string
```

Comment:
A comment about what the version string looks like would be useful.
Is it usually like "5.0.5" or "v5.0.5" or?
That will be helpful for developers who make use of this method.

Maybe we should remove any "v" at the front, and always just return the 3-part semver version n.n.n - then it won't matter in future, if ocis gives the version with "v" or without "v".

Reply:
capability endpoint doesn't have v before the product version

Reply:
ok, then they probably will not randomly add a "v" to the front.